start-all.sh

```diff
@@ -90,6 +90,8 @@ DB_IP="$(sudo docker inspect --format '{{ .NetworkSettings.IPAddress }}' aprom)"
 curl -XPOST -i http://localhost:3000/api/datasources \
      --data-binary '{"name":"prometheus", "type":"prometheus", "url":"'"http://$DB_IP:9090"'", "access":"proxy", "basicAuth":false}' \
      -H "Content-Type: application/json"
-curl -XPOST -i http://localhost:3000/api/dashboards/db --data-binary @./grafana/scylla-dash.json -H "Content-Type: application/json"
-curl -XPOST -i http://localhost:3000/api/dashboards/db --data-binary @./grafana/scylla-dash-per-server.json -H "Content-Type: application/json"
-curl -XPOST -i http://localhost:3000/api/dashboards/db --data-binary @./grafana/scylla-dash-io-per-server.json -H "Content-Type: application/json"
+for v in 1.5; do
```

Comment:
This limit the dashboard the users sees to 1.5, without an ability to bypass and see all dashboards.
Either we extract the version to an external file and have a different value per version, or we always include all dashboards.